static/css/around-map.css

```diff
@@ -44,6 +44,7 @@ main {
   background-color: #fefefe;
   cursor: pointer;
   margin-top: 2rem;
+  border: 0.2rem solid #fefefe;
 }
 .card:hover {
   box-shadow: 0 1rem 2rem 0 rgba(0,0,0,0.2);
```

Comment:
spaces after commas :p

Reply:
I have to install a plugin to see this fucking invisible spaces (if you have a name ...)

Reply:
I just meant `box-shadow: 0 1rem 2rem 0 rgba(0, 0, 0, 0.2);` ;)